python/ray/serve/_private/logging_utils.py

```diff
@@ -262,6 +347,12 @@ def record_factory(*args, **kwargs):
     if logging_config.enable_access_log is False:
         file_handler.addFilter(log_access_log_filter)
 
+    # Redirect print, stdout, and stderr to Serve logger.
+    if not RAY_SERVE_LOG_TO_STDERR:
+        builtins.print = redirected_print
+        sys.stdout = StreamToLogger(logger, logging.INFO)
+        sys.stderr = StreamToLogger(logger, logging.ERROR)
```

Comment:
This should probably be `INFO` not `ERROR` (stderr is commonly used for logging user-facing output)